### DISCUSSION TITLE
PA2 Grading program problem

### DISCUSSION THREAD

**Discussion by idev2580** [ANLAB-KAIST/KENSv3#83] [Project Q&A 2. Transport Layer (TCP)] [answered]
Hello, before start PA#3, I started to fix my PA#2 because it actually doesn't worked well.
But, when I fix it more and more, the result in Wireshark improved(No Duplicated ACK, No unseen Segments, etc.), however, my scores dropped.
As an example, my program successfully got only three connections at TestAccept_Backlog1, but grading program says there was no sockets successfully connected. But in wireshark, they definitely connected successfully.
What should I do? I've worked this for several days but I couldn't found answer.

**Comment by Pusnow** [comment, marked as answer]
The Wireshark only shows info of packets in *wires*.

You should check the system call parts (return values, return timing, etc.), which are not shown in Wireshark.

If you have trouble solving the problem, please send email to TA (cs341.ta.pj@kaist.ac.kr) with your source codes.
